pkgs/applications/networking/deck/default.nix

```diff
@@ -0,0 +1,40 @@
+{ buildGoModule, lib, installShellFiles, fetchFromGitHub }:
+
+buildGoModule rec {
+  pname = "deck";
+  version = "1.22.0";
+  sha = "7447a09";
```

Comment:
What is this for?

Reply:
Is it standard to have a `pname`, `version` in `buildGoModule`?

The `sha` variable will be pass to `ldflags`.

Reply:
> Is it standard to have a `pname`, `version` in `buildGoModule`?

By looking at other derivations, I see that it is a common practice.

Reply:
> The `sha` variable will be pass to `ldflags`.

It is a bit confusing at first, you shouldn't make it a `buildGoModule` attribute in that case.

It's better to wrap it in a `let/in` clause before `buildGoModule` so we can immediately see what are the relevant attributes.

Reply:
Thanks for the advise. Let me update this part.